randovania/gui/widgets/item_tracker_widget.py

```diff
@@ -180,9 +190,11 @@ def update_state(self, inventory: Inventory):
 
                 if isinstance(label, TrackerItemImage):
                     fields = {"amount": amount, "capacity": capacity, "max_capacity": max_capacity}
-                    value_target = element.minimum_to_check
                     value = fields[element.field_to_check.value]
-                    satisfied = max_capacity == 0 or value >= value_target
+                    satisfied = max_capacity == 0 or (
+                        value >= element.minimum_to_check
+                        and (element.maximum_to_check == -1 or value <= element.maximum_to_check)
```

Comment:
I'd use either None or `math.inf`? Though the later is a float...

Reply:
yeah I basically just really didn't want to make the type anything other than int